Kickstarter-iOS/ViewModels/RootViewModel.swift

```diff
@@ -260,8 +263,8 @@ internal final class RootViewModel: RootViewModelType, RootViewModelInputs, Root
 
     let updateBadgeValueOnLifecycleEvents = activityViewControllerIndex
       .takeWhen(lifecycleEvents)
-      .map { index in
-        (activitiesBadgeValue(with: AppEnvironment.current.application.applicationIconBadgeNumber), index)
+      .map { index -> (Int?, RootViewControllerIndex) in
```

Comment:
Had to help the compiler along a few times here because of the merged Signal being a mix of `Int` and `Int?`